examples/Example8_BinarySendReceive/Example8_BinarySendReceive.ino

```diff
@@ -0,0 +1,145 @@
+// Copyright 2023 Blues Inc. All rights reserved.
+//
+// Use of this source code is governed by licenses granted by the
+// copyright holder including that found in the LICENSE file.
+//
+// This example exercises the Notecard's ability to send and receive a binary
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <Notecard.h>
+
+// Note that both of these definitions are optional; just prefix either line
+// with `//` to remove it.
+// - Remove txRxPinsSerial if you wired your Notecard using I2C SDA/SCL pins
+//   instead of serial RX/TX
+// - Remove usbSerial if you don't want the Notecard library to output debug
+//   information
+
+// #define txRxPinsSerial Serial1
+#define usbSerial Serial
+
+// This is the unique Product Identifier for your device
+#ifndef PRODUCT_UID
+#define PRODUCT_UID "" // "com.my-company.my-name:my-project"
+#pragma message "PRODUCT_UID is not defined in this example. Please ensure your Notecard has a product identifier set before running this example or define it in code here. More details at https://dev.blues.io/tools-and-sdks/samples/product-uid"
+#endif
+
+#define myProductID PRODUCT_UID
+Notecard notecard;
+
+// One-time Arduino initialization
+void setup()
+{
+    // Set up for debug output (if available).
+#ifdef usbSerial
+    // If you open Arduino's serial terminal window, you'll be able to watch
+    // JSON objects being transferred to and from the Notecard for each request.
+    usbSerial.begin(115200);
+    const size_t usb_timeout_ms = 3000;
+    for (const size_t start_ms = millis(); !usbSerial && (millis() - start_ms) < usb_timeout_ms;)
+        ;
+    notecard.setDebugOutputStream(usbSerial);
+#endif
+
+    // Initialize the physical I/O channel to the Notecard
+#ifdef txRxPinsSerial
+    notecard.begin(txRxPinsSerial, 9600);
+#else
+    notecard.begin();
+#endif
+
+    // Configure the productUID, and instruct the Notecard to stay connected to
+    // the service.
+    J *req = notecard.newRequest("hub.set");
+    if (myProductID[0])
+    {
+        JAddStringToObject(req, "product", myProductID);
+    }
+    JAddStringToObject(req, "mode", "continuous");
+    notecard.sendRequestWithRetry(req, 5); // 5 seconds
+
+    // Reset the state of the Notecard's binary data store to a known value.
+    NoteBinaryReset();
+}
+
+// In the Arduino main loop which is called repeatedly, add outbound data every
+// 30 seconds
+void loop()
+{
+    // Stop the demo after five iterations to conserve data
+    static unsigned event_counter = 0;
+    if (++event_counter > 5)
+    {
+        while(1);
+    }
+
+    // Send data to the Notecard storage and pull it back
+    {
+        /////////////////////////////////////////////////
+        // Transmit that beautiful bean footage
+        /////////////////////////////////////////////////
+        char data[64] = "https://youtu.be/0epWToAOlFY?t=21";
```

Comment:
For the record I think it's a better example to include the binary file (e.g. `example_image.cpp`) but whatevs.

Reply:
I will update the record accordingly.